spoken/settings.py

```diff
@@ -114,6 +118,7 @@
             ],
         },
     },
+
```

Comment:
+ sign looks added in TEMPLATES dictionary. Please check that.

Reply:
Its a new line which got added. Deleting it.

Reply:
ok